src/util/system.cpp

```diff
@@ -399,6 +386,23 @@ std::optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) co
     return std::nullopt;
 }
 
+fs::path ArgsManager::GetFileArg(std::string pathlike_arg, const fs::path& pathDefault) const
```

Comment:
`pathlike_arg` is in snake case notation, `pathDefault` is in camel case notation.

Reply:
Right. Left comment in the PR this PR builds on: https://github.com/bitcoin/bitcoin/pull/24265#pullrequestreview-874013646

Reply:
Addressed in c71437c93611094aa57eb7da0e166159ff128bdd